lib/chef-dk/policyfile_lock.rb

```diff
@@ -29,7 +29,7 @@ class InstallReport
       attr_reader :ui
       attr_reader :policyfile_lock
 
-      def initialize(ui: ui, policyfile_lock: nil)
+      def initialize(ui: ui(), policyfile_lock: nil)
```

Comment:
`ui()` just calls the `attr_reader` method from above, correct?  So won't that always be nil during initialization since it is an instance method?